src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/Users/DeleteCandidate/DeleteCandidateCommandHandler.cs

```diff
@@ -18,6 +18,7 @@
 using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
+using SFA.DAS.FindAnApprenticeship.InnerApi.FindApprenticeApi.Requests;
 
 namespace SFA.DAS.FindAnApprenticeship.Application.Commands.Users.DeleteCandidate
 {
```

Comment:
Not an issue with this PR but the handler shouldn't be a record, it should be a class.

Reply:
Replaced to class instead of record type